vsc_pdf_template_transformer/lib/models/tpl_text.dart

```diff
@@ -66,4 +69,16 @@ class TplText implements wb.WidgetBuilder {
     print('--- value style: ${value.text.style!.color!.toHex()} ------');
     return value;
   }
+
+  String? _evaluateInput(String? text, Map<String, dynamic> data) {
+    if (text != null) {
+      final evaluator = const ExpressionEvaluator();
+      dynamic res;
+      final context = data;
```

Comment:
This is where you sould build the context as:
```dart
final context = {
   data: data,
   ... other things like helper functions to come later...
}
```

Expressions referencing the data will need to be written as `data.first_name`, for example. We need the `data` distinction because other things will be in the context later - like functions and maybe some contextual information.